docs/advanced-features/athena.md

```diff
@@ -1,91 +1,317 @@
+    ResultConfiguration={'OutputLocation': ATHENA_URL+'/queries/'})
+stat(sqe)
+```
+
+## II. Granting Access to Athena
+
+By default, Quilt runs with very conservative permissions that do not allow access to [Amazon Athena](https://docs.aws.amazon.com/athena/latest/ug/what-is.html). To enable Athena SQL queries by your Quilt users, you must:
+
+1. Create a new Athena policy.
+
+The standard [AmazonAthenaFullAccess](https://console.aws.amazon.com/iam/home#/policies/arn:aws:iam::aws:policy/AmazonAthenaFullAccess) policy is more permissive than necessary.  For production usage, we recommend creating a policy limited to only the above Database:
+<!--pytest-codeblocks:cont-->
+
+
+```python
+# https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/iam.html#IAM.ServiceResource.create_policy
+# https://docs.aws.amazon.com/athena/latest/ug/workgroups-access.html
+
+# TODO: Replace aws-athena-query-results-*
+
+AthenaQuiltAccess={
+            "Version": "2012-10-17",
+            "Statement": [
+                {
+                    "Effect": "Allow",
+                    "Action": [
+                        "athena:*"
+                    ],
+                    "Resource": [
+                        "*"
+                    ]
+                },
+                {
+                    "Effect": "Allow",
+                    "Action": [
+                        "glue:GetDatabase",
+                        "glue:GetDatabases",
+                        "glue:CreateTable",
+                        "glue:DeleteTable",
+                        "glue:UpdateTable",
+                        "glue:GetTable",
+                        "glue:GetTables",
+                    ],
+                    "Resource": [
+                        "*"
+                    ]
+                },
+                {
+                    "Effect": "Allow",
+                    "Action": [
+                        "s3:GetBucketLocation",
+                        "s3:GetObject",
+                        "s3:ListBucket",
+                        "s3:ListBucketMultipartUploads",
+                        "s3:ListMultipartUploadParts",
+                        "s3:AbortMultipartUpload",
+                        "s3:CreateBucket",
+                        "s3:PutObject",
+                        "s3:PutBucketPublicAccessBlock"
+                    ],
+                    "Resource": [
+                        ARN_ATHENA
+                    ]
+                },
+
+            ]
+}
+try: 
+    AthenaQuiltPolicy = IAM.create_policy(
+        PolicyName='AthenaQuiltAccess',
+        PolicyDocument=json.dumps(AthenaQuiltAccess),
+        Description='Minimal Athena Access policy for Quilt'
+    )
+    print(AthenaQuiltPolicy)
+except:
+    print("Policy AthenaQuiltAccess already exists")    
+```
+
+    Policy AthenaQuiltAccess already exists
+
+
+2. Attach this policy to your CloudFormation stack.
+ 
+This needs to be done manually by your AWS Administrator:
```

Comment:
I recently realized, this "ManagedUserRoleExtraPolicies" field emptied every deploy :(

Reply:
That's only because our deployment script, will fix it.

Reply:
I fixed it.

Reply:
Replace ellipsis